src/System.Windows.Forms/src/System/Windows/Forms/Control.ControlVersionInfo.cs

```diff
@@ -172,6 +177,9 @@ private FileVersionInfo GetFileVersionInfo()
 
                 return _versionInfo;
             }
+
+            //[UnconditionalSuppressMessage("SingleFile", "IL3000")]
```

Comment:
Is this commented out code needed?
```suggestion
            // [UnconditionalSuppressMessage("SingleFile", "IL3000")]
```

Reply:
Woops, good catch! That shouldn't be necessary now.